Reply/app/src/main/res/values-night/themes.xml

```diff
@@ -19,24 +19,40 @@
 
     <style name="Theme.Reply.Dark" parent="Theme.Reply">
         <!--Color-->
-        <item name="colorPrimary">@color/reply_blue_200</item>
-        <item name="colorPrimaryVariant">@color/reply_blue_300</item>
-        <item name="colorSecondary">@color/reply_orange_300</item>
-        <item name="colorSecondaryVariant">@color/reply_orange_300</item>
-
-        <item name="android:colorBackground">@color/reply_black_900</item>
-        <item name="colorSurface">@color/reply_black_800</item>
-        <item name="colorPrimarySurfaceVariant">?attr/colorSurface</item>
-        <item name="colorError">@color/reply_red_200</item>
-
-        <item name="colorOnPrimary">@color/reply_black_900</item>
-        <item name="colorOnSecondary">@color/reply_black_900</item>
-        <item name="colorOnBackground">@color/reply_white_50</item>
-        <item name="colorOnSurface">@color/reply_white_50</item>
-        <item name="colorOnError">@color/reply_black_900</item>
+        <item name="colorPrimary">@color/reply_dark_primary</item>
+        <item name="colorOnPrimary">@color/reply_dark_on_primary</item>
+        <item name="colorPrimaryContainer">@color/reply_dark_primary_container</item>
+        <item name="colorOnPrimaryContainer">@color/reply_dark_on_primary_container</item>
```

Comment:
i think we should also be setting colorPrimaryInverse - was that not generated?

Reply:
That was not generated.